kotlin-xml-builder/src/main/kotlin/org/redundent/kotlin/xml/PrintOptions.kt

```diff
@@ -47,7 +47,13 @@ class PrintOptions(
 	 *
 	 * if true "'" becomes &#39;
 	 */
-	val useCharacterReference: Boolean = false
+	val useCharacterReference: Boolean = false,
+
+	/**
+	 * Changes the indent for new lines when [pretty] is enabled. The option has no effect when
+	 * [print] is set to `false`. The default uses one tab `\t`.
+	 */
+	val indent: String = "\t"
```

Comment:
I added the option to the end for source code compatibility in case someone calls the constructor without named arguments.